DigitalLearningSolutions.Web/Services/RegistrationService.cs

```diff
@@ -200,16 +203,33 @@ IGroupsService groupsService
 
             try
             {
+                var emailUpdate = new PossibleEmailUpdate
+                {
+                    OldEmail = userDataService.GetCentreEmail(userId, internalDelegateRegistrationModel.Centre),
+                    NewEmail = delegateRegistrationModel.CentreSpecificEmail,
+                    NewEmailIsVerified = false,

```

Comment:
`NewEmailIsVerified` will be true if:
* `delegateRegistrationModel.CentreSpecificEmail` matches `userEntity.UserAccount.PrimaryEmail` and `userEntity.UserAccount.EmailVerified` is not null, 
* `delegateRegistrationModel.CentreSpecificEmail` matches any verified `UserCentreDetails` for the user

so you'll need to use `EmailVerificationService.AccountEmailIsVerifiedForUser` here